src/diffusers/utils/peft_utils.py

```diff
@@ -129,7 +129,7 @@ def get_peft_kwargs(rank_dict, network_alpha_dict, peft_state_dict, is_unet=True
         rank_pattern = dict(filter(lambda x: x[1] != r, rank_dict.items()))
         rank_pattern = {k.split(".lora_B.")[0]: v for k, v in rank_pattern.items()}
 
-    if network_alpha_dict is not None:
+    if network_alpha_dict is not None and len(network_alpha_dict) > 0:
```

Comment:
```suggestion
    if network_alpha_dict:
```

Wouldn't this be sufficient?

Reply:
I think that we should avoid python magic and make sure we test explicit checks :D

Reply:
I wouldn't consider this "magic", even PEP8 recommends it:

> For sequences, (strings, lists, tuples), use the fact that empty sequences are false:
> 
> \# Correct:
> if not seq:
> if seq:
> 
> \# Wrong:
> if len(seq):
> if not len(seq):